README.md

```diff
@@ -60,7 +60,9 @@ Input                     | Description
 
 MATLAB exits with exit code 0 if the specified script, function, or statement executes successfully without error. Otherwise, MATLAB terminates with a nonzero exit code, which causes the build to fail. To ensure that the build fails in certain conditions, use the [`assert`](https://www.mathworks.com/help/matlab/ref/assert.html) or [`error`](https://www.mathworks.com/help/matlab/ref/error.html) functions.
 
-When you use this action, all of the required files must be on the MATLAB search path.
+When you use this action, all of the required files must be on the MATLAB search path. If your script or function is located in a folder in the root of your repository, you can use the [`addpath`](https://www.mathworks.com/help/matlab/ref/addpath.html), [`cd`](https://www.mathworks.com/help/matlab/ref/cd.html), or [`run`](https://www.mathworks.com/help/matlab/ref/run.html) functions to ensure that the folder is on the path when the script or function is invoked. For example, to run `myscript.m` in a folder `myfolder` in the root of the repository, you can specify `command` like this: 
```

Comment:
I first read this as "If your script or function is located in the root folder..."

Would it be clearer to say "If your script or function is not located in the root of your repository, ..." ?

Reply:
Yes, I think this conveys the message more clearly. Updated the paragraph.